### PR TITLE
change to nexmo sip

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,72 +254,33 @@ repo on GitHub.
...
 
   For example, the new lines in the config.json file should look like this:
 ```json
        "SIP": {
-         "sipUri" : "sip:yourapp145617992434@phone.plivo.com",
-         "sipUsername" : "yourapp145617992434@phone.plivo.com",
-         "sipPassword" : "sip:yourpassword",
+         "sipUri" : "sip:phoneumber@sip.nexmo.com",
+         "sipUsername" : "nexmoApiKey",
+         "sipPassword" : "nexmoPassword",
```
Comment (jeffswartz) [inline README.md]: Please change `"nexmoPassword"` to `"nexmo API secret"`.

File: server/serverMethods.js [hunk 2]
```diff
@@ -760,12 +711,6 @@ function ServerMethods(aLogLevel, aModules) {
         if (!dialedNumberInfo || dialedNumberInfo.googleIdToken !== googleIdToken) {
           return aRes.status(400).send(new ErrorInfo(400, 'Unknown phone number.'));
         }
-        if (dialedNumberInfo.uuid) {
-          var params = {
-            call_uuid: dialedNumberInfo.uuid,
-          };
-          plivoClient.hangup_call(params);
```
Comment (jeffswartz) [inline server/serverMethods.js]: If you click the hang-up button before the callee answers the phone, the call keeps dialing. Is there a way to disconnect (or hang up) the SIP call if the callee hasn't answered? (Calling force disconnect on the SIP connection doesn't hang up the call.)

Comment (jeffswartz) [reply server/serverMethods.js]: One other thing, @msach22: When I run `npm install` on this branch, I see a lot of diffs in the package-lock.json file.

Comment (msach22) [reply server/serverMethods.js]: @jeffswartz updated the package-lock.json file!

**Pull request merged**